src/main/java/org/prebid/server/floors/PriceFloorFetcher.java

```diff
@@ -197,9 +202,12 @@ private static int resolveMaxRules(Long accountMaxRules) {
 
     private Long cacheTtlFromResponse(HttpClientResponse httpClientResponse, String fetchUrl) {
         final String cacheMaxAge = httpClientResponse.getHeaders().get(HttpHeaders.CACHE_CONTROL);
```

Comment:
Bad variable name. It's not cache max age, it's cache-control header value.

Reply:
Done